changelog/7063.bugfix.md

```diff
@@ -0,0 +1,4 @@
+Update the spec of ``POST /model/test/intents`` and add tests for cases when JSON is provided.
+
+Fix the incorrect temporary file extension for the data that gets extracted from the payload provided
+in the body of ``POST /model/test/intents`` request.
```

Comment:
```suggestion
Update the spec of `POST /model/test/intents` and add tests for cases when JSON is provided.

Fix the incorrect temporary file extension for the data that gets extracted from the payload provided
in the body of `POST /model/test/intents` request.
```

Reply:
I still write RST instead of Markdown :)

Reply:
you're so 2020 😆